test/interval_tests/trig.jl

```diff
@@ -3,227 +3,223 @@
 using IntervalArithmetic
 using Test
 
-setprecision(Interval, 128)
-setprecision(Interval, Float64)
-
 @testset "sin" begin
-    @test sin(@interval(0.5)) == Interval(0.47942553860420295, 0.47942553860420301)
-    @test sin(@interval(0.5, 1.67)) == Interval(4.7942553860420295e-01, 1.0)
-    @test sin(@interval(1.67, 3.2)) == Interval(-5.8374143427580093e-02, 9.9508334981018021e-01)
-    @test sin(@interval(2.1, 5.6)) == Interval(-1.0, 0.863209366648874)
-    @test sin(@interval(0.5, 8.5)) == Interval(-1.0, 1.0)
-    @test sin(@floatinterval(-4.5, 0.1)) == Interval(-1.0, 0.9775301176650971)
-    @test sin(@floatinterval(1.3, 6.3)) == Interval(-1.0, 1.0)
-
-    @test sin(@biginterval(0.5)) ⊆ sin(@interval(0.5))
-    @test sin(@biginterval(0.5, 1.67)) ⊆ sin(@interval(0.5, 1.67))
-    @test sin(@biginterval(1.67, 3.2)) ⊆ sin(@interval(1.67, 3.2))
-    @test sin(@biginterval(2.1, 5.6)) ⊆ sin(@interval(2.1, 5.6))
-    @test sin(@biginterval(0.5, 8.5)) ⊆ sin(@interval(0.5, 8.5))
-    @test sin(@biginterval(-4.5, 0.1)) ⊆ sin(@interval(-4.5, 0.1))
-    @test sin(@biginterval(1.3, 6.3)) ⊆ sin(@interval(1.3, 6.3))
+    @test sin(Interval(0.5)) ≛ Interval(0.47942553860420295, 0.47942553860420301)
```

Comment:
I think tests should have `interval` or `..` (the one that doesn't do black magic) on the LHS and `Interval` on the RHS. The reason for this is that you want to use on the left a constructor that does the checks that has to be done (e.g. check that errors when an invalid input is given), but at the same time at the RHS you want the ground truth as given without any processingn, otherwise two errors on both sides could cancel out each others. (ITF1788.jl generates the tests with this logic)
Finally, I think in testing === should be preferred.